tests/test_utils.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+
+from baseframe import _localized_country_list_inner
+from .fixtures import TestCaseBaseframe
+
+
+class TestUtils(TestCaseBaseframe):
+    def test_localized_country_list(self):
+        countries = _localized_country_list_inner('en')
+        assert dict(countries)['DE'] == u"Germany"
+        countries = _localized_country_list_inner('de')
+        assert dict(countries)['DE'] == u"Deutschland"
+        countries = _localized_country_list_inner('es')
+        assert dict(countries)['DE'] == u"Alemania"
+        countries = _localized_country_list_inner('hi')
+        assert dict(countries)['DE'] == u"जर्मनी"
```

Comment:
1. You need a test for the outer function, which means you have to simulate locale by making a test request with a custom Accept header. Don't test the inner -- it's not meant to be used anywhere outside the outer function.

2. Test sorting by country name. Pick one country, say `DE` as above, and see whether it indexes before or after another country in various locales.